packages/editor-web-component/README.md

```diff
@@ -4,7 +4,7 @@ This is an early version of the web component wrapping the [Serlo Editor](https:
 
 ## Installation and usage
 
-1. `yarn install @serlo/editor-web-component`
+1. `yarn add @serlo/editor-web-component`
 2. Define an HTML element with the id `serlo-root` in your file where you render the web component
 
 ```JSX
```

Comment:
would suggest adding docs for the event here. Maybe for later, we'll have to add more documentation anyways sooner or later when more properties follow (e.g. definition of extensions from the outside)